src/content/docs/fr/tutorial/5-astro-api/4.mdx

```diff
@@ -95,7 +95,7 @@ Les personnes peuvent s'abonner à votre flux dans un lecteur de flux et recevoi
     import { defineConfig } from "astro/config";
 
     export default defineConfig({
-      site: "https://exemple.com"
+      site: "https://mon-site-de-blog.netlify.app"
```

Comment:
```suggestion
      site: "https://example.com"
```
I'm going to ask our French translators for advice here! Normally "example.com" is a reserved URL for documentation etc. and it is recommended to use instead of other specific domain names.

But, I'm not sure whether the translated version is *also* a protected domain name specifically for testing and documentation! What do we think??

Reply:
Owwh good catch @borelkoumo ! Yes @sarah11918, it's a translated URL, but it links to a fraudulent site or whatever. So it's better to keep `example.com`!

Reply:
Hi @sarah11918 and @thomasbnt 
I'm excited to use Astro, and to make my very first contribution to this framework.
I also think it's better to keep `example.com`, which is a well known placeholder